### PR TITLE
Added temporary make target to build and push operator and catalog image

### PR DESCRIPTION
**Pull request by arjunrn**
Signed-off-by: Arjun Naik <anaik@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,3 +3,6 @@ include boilerplate/generated-includes.mk
 .PHONY: boilerplate-update
 boilerplate-update:
 	@boilerplate/update
+
+.PHONY: build-push
```
Comment (2uasimojo) [inline Makefile:7]: Why is this necessary? It's already in [the convention's standard.mk](https://github.com/openshift/osd-metrics-exporter/blob/main/boilerplate/openshift/golang-osd-operator/standard.mk#L172-L174)

Comment (arjunrn) [reply Makefile:7]: Because the `build-push` in the convention only builds and pushes the operator image but not the catalog image.

Comment (arjunrn) [reply Makefile:7]: My bad, looks like it does.
